TKGame/Level Editor Content/Trigger.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame.Level_Editor_Content
+{
+    internal class Trigger
+    {
+        #region Member Variables
+        private static readonly int screenWidth = 1600;
+        private static readonly int screenHeight = 900;
+        private static readonly int xBuffer = 100;
+        private static readonly int yMax = 825;
+        private static readonly int yMin = 600;
+        public Rectangle rectangle { get; set; }
+        public Texture2D texture { get; set; }
+
+        public string leftStage { get; set; }
```

Comment:
This will most likely be refactored out to instead have a dict of adjacent stages for vertically entered levels later on